specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -0,0 +1,241 @@
+              <Input.Text
+                className="h-full"
+                isReadOnly={isReadOnly}
+                value={value ?? ''}
+                onValueChange={(value): void =>
+                  handleChanged(
+                    {
+                      value: value.length === 0 ? undefined : value,
+                      fields,
+                    },
+                    index
+                  )
+                }
+              />
+            </div>
+            <span className="-ml-2" />
+            {trimmedFieldsLength > 0 && isExpanded ? (
+              <Button.Danger
+                onClick={(): void => {
+                  handleChange(removeItem(formatter.definition.fields, index));
+                }}
+              >
+                {resourcesText.deleteDefinition()}
+              </Button.Danger>
+            ) : null}
+          </div>
+          {isExpanded ? (
+            <span>
+              {index === 0
+                ? resourcesText.elseConditionDescription()
+                : resourcesText.conditionDescription()}
+            </span>
+          ) : null}
+        </Label.Block>
+      )}
+      {isExpanded || !hasCondition ? null : fields.length === 0 ? (
+        <Button.Small
+          onClick={(): void => {
+            handleToggle();
+            handleChanged(
+              {
+                value,
+                fields: [
+                  {
+                    separator: localized(' '),
+                    aggregator: undefined,
+                    formatter: undefined,
+                    fieldFormatter: undefined,
+                    field: undefined,
+                  },
+                ],
+              },
+              index
+            );
+          }}
+        >
+          {resourcesText.addField()}
+        </Button.Small>
+      ) : (
+        fields.map((field, index) => (
+          <p key={index}>
+            {field.separator === undefined ? '' : field.separator}
+            {field.field === undefined ? '' : field.field[0].label}
+          </p>
+        ))
+      )}
+      {isExpanded || !hasCondition ? (
+        <Fields
+          fields={[
+            fields,
+            (fields): void => handleChanged({ value, fields }, index),
+          ]}
+          table={table}
+        />
+      ) : null}
+      <span className="-ml-2 flex-1" />
+      <div className="inline-flex">
+        {trimmedFieldsLength === 1 || isExpanded ? null : (
+          <Button.Icon
+            icon="trash"
+            title={resourcesText.deleteDefinition()}
+            onClick={(): void => {
+              handleChange(removeItem(formatter.definition.fields, index));
+            }}
+          />
+        )}
+      </div>
+      <div className="flex">
+        {isExpanded ? <span className="-ml-2 flex-1" /> : null}
+        {hasCondition ? (
+          <Button.Icon
+            icon={isExpanded ? 'chevronUp' : 'chevronDown'}
+            title="showConditionalField"
```

Comment:
good catch @pashiav!

Reply:
this doesn't look resolved - can you use a localized title, not a camelCased string?